DEVELOPING.md

```diff
@@ -0,0 +1,21 @@
+# Developer's Guide
+
+We use Visual Studio Code for developing LoopBack and recommend the same to our contributors.
+
+While this package supports both Node.js 6.x and 8.x versions, you will need Node.js 8.x (or newer) for the best development experience in VS Code.
+
+## VSCode setup
+
+Install the following extensions:
+
+ - [tslint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint)
+ - [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+
+## Development workflow
```

Comment:
Could you update this file to match the latest content in loopback-next-extension-starter please?